evaluations/inkscope-fuzzer_2_dsm-w3f.md

```diff
@@ -14,7 +14,29 @@
 | *0d.* | Docker | <ul><li>[x] </li></ul>| https://github.com/inkscopexyz/inkscope-fuzzer/tree/v0.0.1/.docker / https://github.com/inkscopexyz/inkscope-fuzzer/blob/v0.0.1/book/src/installation.md#using-inkscope-fuzzer-with-docker |  |
 | *0e.* | Article | <ul><li>[x] </li></ul>| https://blog.inkscope.xyz/inkscope-ink-fuzzer | |
 | 1. | Mutation Engine | <ul><li>[ ] </li></ul>| https://github.com/inkscopexyz/inkscope-fuzzer/tree/v0.0.1 | Couldn't run some examples |
-| 2. | Fuzzer | <ul><li>[ ] </li></ul>| https://github.com/inkscopexyz/inkscope-fuzzer/tree/v0.0.1 | Couldn't run some examples. It should generate a file with the results? |
+| 2. | Fuzzer | <ul><li>[ ] </li></ul>| https://github.com/inkscopexyz/inkscope-fuzzer/tree/v0.0.1 | Couldn't run some examples. I didn't fully understand the output file. |
+
+## Evaluation V2
+
+### Output file
+
+I was able to generate the output fine. However, I didn't fully understand the file. I think it could be more human-readable or have a tutorial teaching how to debug and reproduce the error using the output file. The output generated was similar to the [example](https://github.com/inkscopexyz/inkscope-fuzzer/tree/da5f097e151e64cdc4b00ce652016225b1241389?tab=readme-ov-file#-how-to-run-the-fuzzer) on the readme.
+
+### Manual Test
+
+The changes fixed the building problem. However, I'm having trouble running the inkscope-fuzzer on the coinfabrik contracts from the `assert-violation` and `iterators-over-indexing` cases. In both examples, I received the error below.
```

Comment:
Regarding this issue, the error you received is expected behavior.

You are having that error because both contracts do not contain any mutable messages or properties. We have added an explicit error to inform when this happens, which says: 'No executable messages found in the contract' instead of just a panic.

In order to fuzz these contracts, please follow these steps:

1. Create a copy of the example-config.yaml file.
2. Set the only_mutable field to false in the copied configuration file.
3. Execute the fuzzer while passing the copied configuration file as a parameter:

```bash
inkscope-fuzzer /path/to/contract.contract fuzz --config /path/to/copied-config.yaml
```
By setting only_mutable to false, the fuzzer will consider all messages and properties in the contract, including non-mutable ones